tests/telemetry/events/event_utils.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from tests.common.utilities import wait_until
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+
+def backup_monit_config(duthost):
+    logger.info("Backing up monit config files")
+    duthost.shell("cp -f /etc/monit/monitrc /tmp/")
```

Comment:
>tmp [](http://example.com/codeflow?start=45&length=3)

Prefer home dir

Reply:
I will try changing from using /tmp to /home

Reply:
Done